services/app-web/src/components/SubmissionSummarySection/RateDetailsSummarySection/SingleRateSummarySection.tsx

```diff
@@ -182,23 +197,60 @@ export const SingleRateSummarySection = ({
     }, [getKey, getBulkDlURL, formData])
     // END bulk download logic
 
+    const [unlockRate, { loading: unlockLoading }] = useUnlockRateMutation()
+
+    const handleUnlockRate = async () => {
+        try {
+            const { data } = await unlockRate({
+                variables: {
+                    input: {
+                        rateID: rate.id,
+                        unlockedReason: '',
+                    },
+                },
+            })
+
+            if (data?.unlockRate.rate) {
+                // navigate('') TODO
```

Comment:
nit: Is there a redirect here? From the AC, there is no mention of one, and if this follows the same pattern as contracts, it does not redirect the CMS user after unlocking. It might be confusing to leave a comment here suggesting a TODO for navigating elsewhere.

Reply:
Ah yes, I'll remove this!